src/sidebar/lib/contextmenu/contextmenu.js

```diff
@@ -45,11 +45,11 @@ ContextMenu.prototype = {
     const winHeight = window.innerHeight;
 
     const top = this.clickPosY + menuHeight > winHeight ?
-                (winHeight - menuHeight) :
-                this.clickPosY;
+      (winHeight - menuHeight) :
```

Comment:
Please keep the indentation (here and other files).

Reply:
hmm, eslint must have done those.